### PR TITLE
Update command-line usage text

### PR DESCRIPTION
**Pull request by ffoulkes**
- Edited usage text for command-line utilities and daemons. Made name column 30 characters wide.
- Refactored ovs-vctl `usage()` function into several smaller functions.
- Added text for P4 commands.

### CODE REVIEW

**Comment by 5abeel** [comment]
Can you elaborate why all the output strings are made longer with extra space characters in this PR?

File: utilities/ovs-vsctl.c [hunk 1]
```diff
@@ -442,14 +448,40 @@ SSL commands:\n\
 Auto Attach commands:\n\
   add-aa-mapping BRIDGE I-SID VLAN   add Auto Attach mapping to BRIDGE\n\
   del-aa-mapping BRIDGE I-SID VLAN   delete Auto Attach mapping VLAN from BRIDGE\n\
-  get-aa-mapping BRIDGE              get Auto Attach mappings from BRIDGE\n\
-\n\
+  get-aa-mapping BRIDGE              get Auto Attach mappings from BRIDGE\n");
+}
+
+#ifdef P4OVS
```
Comment (5abeel) [inline utilities/ovs-vsctl.c:454]: Will this be defined elsewhere in another commit/PR?

File: utilities/ovs-vsctl.c [hunk 2]
```diff
@@ -458,17 +490,43 @@ Options:\n\
...
     printf("\n\
 Other options:\n\
   -h, --help                  display this help message\n\
   -V, --version               display version information\n");
+}
+
+static void
+usage(void)
+{
+    ovs_usage(program_name);
```
Comment (5abeel) [inline utilities/ovs-vsctl.c:517]: Order seems random. Should this be lexicographically ordered?

File: utilities/ovs-vsctl.c (hunk 1 above)
Comment (ffoulkes) [reply utilities/ovs-vsctl.c:454]: The p4 commands were added in the previous commit.

Based on the conversation at today's meeting, these commands may well be backed out. (In fact, it sounds as if a large part of the first commit may be backed out.) The p4 bridge is an implementation mechanism for multiple pipelines, which we do not yet support.

File: utilities/ovs-vsctl.c (hunk 2 above)
Comment (ffoulkes) [reply utilities/ovs-vsctl.c:517]: I would guess that the author of the changes was trying to preserve the existing order, and that the p4 commands were inserted following the Auto Attach commands because they both have to do with BRIDGEs.

As for the reason for the existing order ... no clue.

**Comment by ffoulkes** [comment]
> Can you elaborate why all the output strings are made longer with extra space characters in this PR?

The first column was widened from 27 to 30 characters. I assume this was because of the number of instances where the text in the first column exceeded its size, resulting in a choppy right-hand column.

If you look at the side-by-side comparisons in `ovsdb-tool` and `ovs-dptl`, I think you'll see that the new text is a little more readable than the old text.

There were also a couple of places where a two-space indent was added. I believe this was for greater consistency among the various tools.

I'm extracting other people's edits from P4-OVS and reapplying them in a separate branch. I don't necessarily know the reasons why they made them. :)

File: utilities/ovs-vsctl.c (hunk 1 above)
Comment (5abeel) [reply utilities/ovs-vsctl.c:454]: OK. Will wait for further commits to remove this code then if we are backing out on some p4 specific changes related to multiple pipelines

**Comment by ffoulkes** [comment]
PR abandoned
